### PR TITLE
DIV-5378 - Added Jenkins test_environment and refactored tests

### PR DESCRIPTION
**Pull request by hosainnet**
# Description

Recent changes in #114 and #113  appear to require an explicit setting of the test_environment variable so that the correct application-${env}.properties file is picked up

Fixes #DIV-5378

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -44,6 +44,10 @@ withPipeline(type, product, component) {
         echo '${product}-${component} checked out'
     }
 
+    before('functionalTest:preview') {
+        env.test_environment = 'aat'
+    }
+
```
Comment (hosainnet) [inline Jenkinsfile_CNP]: We do the same for `functionalTest:preview` below

File: src/integrationTest/java/uk/gov/hmcts/reform/divorce/emclient/EMClientFileDownloadTest.java [hunk 2]
```diff
@@ -23,12 +18,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThat;
 
-@Lazy
 @RunWith(SerenityRunner.class)
-@ComponentScan(basePackages = {"uk.gov.hmcts.reform.divorce.emclient", "uk.gov.hmcts.auth.provider.service"})
-@ImportAutoConfiguration({HttpMessageConvertersAutoConfiguration.class})
-@PropertySource("classpath:application.properties")
-@PropertySource("classpath:application-${env}.properties")
```
Comment (hosainnet) [inline src/integrationTest/java/uk/gov/hmcts/reform/divorce/emclient/EMClientFileDownloadTest.java]: Moved to base

File: Jenkinsfile_CNP (hunk 1 above)
Comment (tchow8) [reply Jenkinsfile_CNP]: ? Isn't this duplicated unless you meant functionalTest:aat?

Comment (hosainnet) [reply Jenkinsfile_CNP]: Ah damn you're right

**Pull request merged**